### PR TITLE
Feature/authentication modal

### PR DESCRIPTION
**Pull request by NancyRike**

### CODE REVIEW

**Comment by NancyRike** [comment]
Shipping

File: src/App.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { NotificationContainer } from "react-notifications";
 import OrderPage from './pages/orderPage/OrderPage'
 import checkoutPage from './pages/CheckoutPage/checkoutPage'
 import ScrollToTop from './components/scrollTop'
+import Shipping from './components/shipping/shipping'
 // import SignIn from './components/Authentication/signIn'
```
Comment (Dverybest) [inline src/App.js]: Remove used code and clean up comments

File: src/App.js [hunk 2]
```diff
@@ -22,6 +23,8 @@ function App() {
           <Route exact path='/contact-us' component={ContactPage} />
           <Route exact path='/order' component={OrderPage}/>
           <Route exact path="/checkout" component={checkoutPage}/>
+          <Route exact path="/shipping" component={Shipping}/>
+
           {/* <Route exact path="/signIn" component={SignIn}/>
```
Comment (Dverybest) [inline src/App.js]: Remove

File: src/components/Authentication/signIn.js [hunk 3]
```diff
@@ -49,8 +49,14 @@ const SignIn = (props) => {
...
           </div>
           <div className="d-flex justify-content-end align-items-rigt">
-            <button onClick={()=>props.setmanageAuth("signUp")} className="btn registerButton">Create Account</button>
-            <button className="btn registerButton activeButton" onClick={handleSignIn}>{props.isLoading?'Authenticating...':'Sign In'}</button>
+            <button onClick={(e)=>{
+              e.preventDefault()
+              props.setmanageAuth("signUp")
+            }} className="btn registerButton">Create Account</button>
+            <button className="btn registerButton activeButton" onClick={handleSignIn}>
+              {/* {props.isLoading?'Authenticating...':'Sign In'} */}
```
Comment (Dverybest) [inline src/components/Authentication/signIn.js:55]: remove comment

File: src/pages/CheckoutPage/checkoutPage.css [hunk 4]
```diff
@@ -45,9 +45,9 @@
     line-height: 24px;
     color: #000000;
 }
-.imageWrapper{
-    /* width: 65%; */
-}
+/* .imageWrapper{
```
Comment (Dverybest) [inline src/pages/CheckoutPage/checkoutPage.css:48]: remove unused code

File: src/pages/CheckoutPage/checkoutPage.js [hunk 5]
```diff
@@ -22,7 +24,13 @@ const CheckoutPage = (props) => {
 
     props.handleDecrementChange(index);
   };
-
+  // const [show, setshow] = useState(false)
+  const [manageAuth, setmanageAuth] = useState('signIn');
+  // handlePageChange = (e, auth)=>{
```
Comment (Dverybest) [inline src/pages/CheckoutPage/checkoutPage.js]: Remove if not used

**Review by Dverybest — CHANGES_REQUESTED** [review]
Looks good, a little clean up required

File: src/pages/CheckoutPage/checkoutPage.js (hunk 5 above)
Comment (NancyRike) [reply src/pages/CheckoutPage/checkoutPage.js]: Fixed

File: src/pages/CheckoutPage/checkoutPage.css (hunk 4 above)
Comment (NancyRike) [reply src/pages/CheckoutPage/checkoutPage.css:48]: Fixed

File: src/components/Authentication/signIn.js (hunk 3 above)
Comment (NancyRike) [reply src/components/Authentication/signIn.js:55]: handled

File: src/App.js (hunk 2 above)
Comment (NancyRike) [reply src/App.js]: fixed

File: src/App.js (hunk 1 above)
Comment (NancyRike) [reply src/App.js]: done

**Review by Dverybest — APPROVED** [review]

**Pull request merged**